chromadb/test/property/test_cross_version_persist.py

```diff
@@ -152,47 +151,47 @@ def switch_to_version(version):
 
     # Load the target version and override the path to the installed version
     # https://docs.python.org/3/library/importlib.html#importing-a-source-file-directly
-    path = get_path_to_version_library(version)
     sys.path.insert(0, get_path_to_version_install(version))
-    spec = spec_from_file_location(module_name, path)
-    assert spec is not None and spec.loader is not None
-    module = module_from_spec(spec)
-    spec.loader.exec_module(module)
-    assert module.__version__ == version
-    sys.modules[module_name] = module
-    return module
+    import chromadb
```

Comment:
This is changing how import works in the process that runs older version. We rely on the path instead of manually constructing it.